### DISCUSSION TITLE
Huawei P8 Lite 2017 (PRA-LX1 with Kirin 655)

### DISCUSSION THREAD

**Discussion by MatteoErr** [kitsuned/PotatoNV#125] [Q&A] [answered]
Hello,
I see this in the suppported devices : 
<html>
<body>

Huawei P8 Lite (2017) | PRA | Kirin 65x (A)
-- | -- | --

</body>
</html>
but my Huawei P8 Lite 2017 has a Kirin 655 CPU (that isn't in the list of supported CPUs) and is PRA-LX1 instead of PRA.
Is my phone comptaible or not ? If not will it be supported soon ?

**Comment by MatteoErr** [comment, marked as answer]
If someone is searching the answer to this question, I have tried it and it worked.

**Reply by jomoreau** [reply]
Hi MatteoErr,
I'd like to do the same as you but I haven't been able to short-circuit the testpoint. My USB cable seems ok because the phone shows on my file manager (Ubuntu) when I plug it, although Hisuite (through Wine) does not detect the connection. I've tried to short the testpoint with tweezers, metal thread, copper wire... with no success. The target device box in PotatoNV shows empty.
How did you manage it?

**Reply by MatteoErr** [reply]
For me it didn't work with tweezers but I used a metal wire. But I did it on Windows, not on Linux. Maybe the problem for you is related to the drivers. Also, to make HiSuite work you have to enable USB debugging (or at least HDB).

**Reply by jomoreau** [reply]
Hi,
Finally, it worked fine on first trial with Windows.
